src/lib/components/modal/LogisticsModal.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
```

Comment:
Acredito que se não for um modal genérico é melhor seguir o mesmo estido no frond ai o caminho ficaria como:
`src/lib/components/logistics/modal/LogisticsModal.svelte

Reply:
corrigido - cee9cc839854addd8dc79734fbbc741139bd56be